src/components/common/item-selector/item-selector.js

```diff
@@ -55,8 +55,9 @@ export const StyledDropdownSelect = styled.div.attrs({
 
 const DropdownSelectValue = styled.span`
   color: ${props =>
-    props.hasPlaceholder && props.inputTheme === 'light' ?
-    props.theme.selectColorPlaceHolderLT :  props.hasPlaceholder
+    props.hasPlaceholder && props.inputTheme === 'light'
```

Comment:
this is also in the previous PR

Reply:
yep, will land one and rebase the other. they are automatically corrected by `lint`